apps/masterbots.ai/lib/helpers/ai-helpers.tsx

```
@@ -42,291 +43,331 @@ export function createPayload(
-              : message.role.replace('system', 'assistant'),
-            content: message.content
-          }) as MessageParam
-      )
-    case 'OpenAI':
-    case 'Perplexity':
+              ? message.role.replace("system", "user")
+              : message.role.replace("system", "assistant"),
+            content: message.content,
+          }) as MessageParam,
+      );
+    case "OpenAI":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)